config/settings.yml

```diff
@@ -90,7 +90,7 @@ aca:
   dental_plan_years_available: ["2018"]
   minimum_primary_contribution_percentage: 50
   minimum_family_contribution_percentage: 30
-  plan_options_available: ["sole_source"]
+  plan_options_available: ["sole_source", "single_carrier", "metal_level", "single_plan"]
   ## acceptable options are ["sole_source", "single_carrier", "metal_level", "single_plan"]
```

Comment:
If they just want vertical carrier choice you should only enable single_carrier.